clustered/client/src/test/java/org/ehcache/clustered/client/BasicClusteredCacheTest.java

```diff
@@ -88,7 +88,7 @@ public void testClusteredCacheTwoClients() throws Exception {
         newCacheManagerBuilder()
             .with(cluster(CLUSTER_URI))
             .withCache("clustered-cache", newCacheConfigurationBuilder(Long.class, String.class,
-                ResourcePoolsBuilder.newResourcePoolsBuilder().heap(100, EntryUnit.ENTRIES)
+                ResourcePoolsBuilder.newResourcePoolsBuilder().heap(100, EntryUnit.ENTRIES).offheap(1, MemoryUnit.MB)
```

Comment:
Right now this test doesn't actually do anything with the offheap since the heap tier is large enough to cache all the values used in the test.  Would probably be good to add a new test that used a very small heap tier so that we definitely need to be invalidating things out of the offheap tier in order to get correct reads.